src/Settings.hs

```diff
@@ -114,3 +114,8 @@ programPath context@Context {..} = do
         install <- (\l -> l == stage || package == ghc) <$> maybeLatest
         let path = if install then installPath package else buildPath context
         return $ path -/- programName context <.> exe
+
+destDir :: Action FilePath
```

Comment:
Seems to be install-related? Shall it be moved to `Settings.Install`?

Why isn't this an oracle -- what if the user changes this variable?